### ISSUE TITLE
[Feature Request]: Arrow buttons to switch between torrents in detail view

### ISSUE THREAD

**Issue by Antonio36996** [VueTorrent/VueTorrent#2106]
### Description

When I'm renaming files I have to get in and out between the torrent list and the detail view.

### Describe the solution you'd like

If I filter and sort i can get a sorted list of the torrents I have to rename, then I can enter into the first torrent detail view, rename the files and then, I should be able to navigate to the next torrent clicking the right arrow from the detail view without exiting.
![image](https://github.com/user-attachments/assets/3cacb103-e1d5-4598-8008-a5839f6ef607)

### Complementary informations

- [ ] Is this feature already implemented in the default WebUI?

**Comment by Larsluph** [comment]
I really like the idea, though the way you described the solution can't be done due to mobile layout using the arrows to scroll through tabs.

I would put them next to the X mark at the top instead.
